### DISCUSSION TITLE
Default page when visiting Github.com

### DISCUSSION THREAD

**Discussion by nrthbound** [community/community#44307] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I would really appreciate if we had the option to change the default page we land on when visiting github.com. I understand the feed and the reasoning behind it, but for me personally, when I come to Github, what I want to do most commonly is just land on my profile so I can visit my repos/issues/PRs. Sometimes hunting down a repo (especially if it's older) can be a pain when you're on the main screen. I'm not saying it should *default* to this behavior, but rather allow us to change it ourselves to whatever we want.

**Comment by radar-blip** [comment]
One option is to bookmark your profile and sign in from there.

**Reply by nrthbound** [reply]
> One option is to bookmark your profile and sign in from there.

Well, of course. However, it becomes an issue when using multiple devices or especially on Mobile where based on the browser, Bookmarks take longer to access than just logging in and typing out the full URL for the profile. I just find the profile view to be more useful as it contains the most relevant repos I find myself working in. Not saying it'll be that way for everyone, but I don't think it'd take them a lot of effort to simply add a way to change it.

**Reply by radar-blip** [reply]
Yes, I know what you mean. I was going to say it depends on what device you're using.
Anyway, was just a suggestion.

**Comment by zshzebra** [comment]
I have been looking out for this. Hopefully we can get some news from GitHub if this is every going to be implemented.

I'm honestly surprised it doesn't exist.
